sbnobj/SBND/CRT/CRTVeto.hh

```diff
@@ -0,0 +1,44 @@
+/**
+ * \class CRTVeto
+ *
+ * \brief Product to store CRT Veto Flags for a given Event
+ *
+ * \author Alex Antonakis (aantonakis@ucsb.edu)
+ *
+ */
+
+#ifndef SBND_CRTVETO_HH
+#define SBND_CRTVETO_HH
+
+#include "larcoreobj/SimpleTypesAndConstants/geo_vectors.h"
+
+namespace sbnd::crt {
+
+  class CRTVeto {
```

Comment:
Ditto here: you could save a single `std::bitset` and use it as a mask rather than rely on 5 (N?) member variables, but that's a stylistic choice more useful if you want to avoid maintaining meaningful enum names.